from0k2bp.tex

```diff
@@ -408,18 +417,19 @@
 \hypertarget{the-vector-pedersen-commitment}{%
 \subsection[The Vector Pedersen
 Commitment]{\texorpdfstring{\protect\hypertarget{anchor-22}{}{}The
-Vector Pedersen
-Commitment}{The Vector Pedersen Commitment}}\label{the-vector-pedersen-commitment}}
+Vector Pedersen Commitment}{The Vector Pedersen Commitment}}\label{the-vector-pedersen-commitment}}
 
 To extend to a more powerful form of the Pedersen commitment already
 defined, we go from:
-\[C=rH + aG\]
+\[C := rH + aG\]
 to:
-\[C=rH + (v_1G_1 + v_2G_2 + \ldots + v_nG_n) = rH + \textbf{vG}\]
-The individual $G_i$s can be constructed using a simple algorithm of the form
+\begin{align}
+C := rH + (v_1G_1 + v_2G_2 + \ldots + v_nG_n) = rH + \textbf{vG} \label{vector-pedersen-commitment-def:1} 
+\end{align}
+The individual $G_i$ generators can be constructed using a simple algorithm of the form
 already mentioned (like, take a $\mathcal{H}(\textrm{encode}(G)||i)$  where $\mathcal{H}$ represents some hash function).
 The opening of this commitment would of course be the random scalar $r$ and
-the components of the vector $\textbf{v}$. I defer to elsewhere, proof
+the components of the vector $\textbf{v}$. I defer a proof to elsewhere
```

Comment:
again, this correction isn't needed

Reply:
Alright maybe this one can be debated, my use of the comma was a bit weird here but tbh this is so minor it's not worth mentioning.

Reply:
Reverted.